server/src/db/migrations/20230202191007_addRatingToReviews.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.table("reviews", (table) => {
+        table.integer("rating").notNullable()
+    })
+}
+
+/**
+ * @param {Knex} knex
+ */
+exports.down = (knex) => {
+    return knex.schema.table("reviews", (table) => {
+        table.dropColumn("rating")
+    })
```

Comment:
I love that you are making a PR to add this, but this is an example of when you should rollback, edit your createReviews migration and just add `rating` to the original.

The idea is: once something is merged into main, it's likely going to be used somewhere else. In that case, you make a new migration to add a missing column. 

in this case though, this reviews table is entirely new, so it isnt yet in `main`. 

Delete this migration, edit your original createReviews to have rating

Reply:
Deleted AddRatingToReviews migration file; updated Reviews migration file to include rating column and changed "review" to "content"